.github/workflows/build.yml

```diff
@@ -33,7 +33,7 @@ jobs:
       # Note the current convention is to use the -S and -B options here to specify source
       # and build directories, but this is only available with CMake 3.13 and higher.
       # The CMake binaries on the Github Actions machines are (as of this writing) 3.12
-      run: cmake $GITHUB_WORKSPACE -DCMAKE_BUILD_TYPE=$BUILD_TYPE
+      run: cmake $GITHUB_WORKSPACE -DCMAKE_BUILD_TYPE=$BUILD_TYPE -DOPTION_USE_WAYLAND=0
```

Comment:
I'd prefer boolean option values ON or OFF rather than 1 or 0, respectively.

The correct (full) syntax would be `-D OPTION_USE_WAYLAND:BOOL={ON|OFF} where `:BOOL` can be omitted.

Note: There may be other existing '1' settings which I'd like to be changed as well. Thanks.

Reply:
Done.